lua/mapvote/server/modules/map_reset.lua

```diff
@@ -0,0 +1,19 @@
+local resetMapTimerName = "CFC_MapVote_ResetMap"
```

Comment:
Seems like the rest of the addon isn't CFC-prefixed

```suggestion
local resetMapTimerName = "MapVote_ResetMap"

```